docs/news.rst

```diff
@@ -1,6 +1,14 @@
 Release Notes
 =============
 
+**1.8.0 (unreleased)**
+
+* **BREAKING CHANGE:** Moved cloud testing provider credentials into separate
+  files for improved security.
```

Comment:
Absolutely, my pleasure! 😁 

Thanks for all the time and effort you put in your excellent plugins as well! 👍